src/main/java/com/schedule/scheduleshare/domain/entity/Schedule.java

```diff
@@ -0,0 +1,23 @@
+package com.schedule.scheduleshare.domain.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Date;
+
+@AllArgsConstructor
+@Data
+public class Schedule {
+
+    private int id;
+
+    private Date register_date;
+
+    private String title;
+
+    private String location;
+
+    private Date schedule_date;
```

Comment:
カラムの順に定義しないといけませんね。
あと、Javaの変数名の命名規則（キャメルケース）に従いましょう。
```suggestion
    private Date registerDate;

    private String title;

    private Date scheduleDate;

    private String location;
```